apps/betterangels/package.json

```diff
@@ -82,7 +82,8 @@
     "react-native-paper": "*",
     "dotenv": "*",
     "@react-native-picker/picker": "*",
-    "react-native-image-zoom-viewer": "*"
+    "react-native-image-zoom-viewer": "*",
```

Comment:
Still need so remove it from the `package.json` can be done by manually removing it or running `yarn nx run betterangels:sync-deps`